src/utils/tests/score.test.ts

```diff
@@ -8,11 +8,15 @@ import {
 } from "../score";
 
 test("should format a positive cp score correctly", () => {
-  expect(formatScore({ type: "cp", value: 50 })).toBe("+0.50");
+  expect(formatScore({ type: "cp", value: 50 })).toBe("+0.5");
 });
 
 test("should format a negative cp score correctly", () => {
-  expect(formatScore({ type: "cp", value: -50 })).toBe("-0.50");
+  expect(formatScore({ type: "cp", value: -50 })).toBe("-0.5");
+});
+
+test("should format a large cp score correctly", () => {
+  expect(formatScore({ type: "cp", value: 12345.67 })).toBe("+99.0");
```

Comment:
a) centipawn values are only integers
b) while the conversion might be useful for lichess it isn't so much on a local gui, Stockfish displays values up to 200 for centipawns and we have a nice relation between the distance to zero for tablebase position https://github.com/official-stockfish/Stockfish/commit/def296670dbde6cfad446a735f37f25cfe9df6a2 it be unfortunate to lose this information.

Reply:
a) Good point, I'll change that test to use a sensible value. I'm still learning the engine interface stuff as I go
b) Afaik, the truncated values will only be used on the eval bar, which is too narrow to display arbitrarily sized values without overflowing. The exact value of the eval bar is displayed by a tooltip

Reply:
b) ah okay if it is only the evalbar then it is fine, I had the fear that it might be reused for the engine evaluation display

Reply:
I just double checked, and there is also some truncation (in my branch, but also upstream) on the summary to the right of the engine component that displays the evaluation of the first variation
The space here is wide enough to use the full value without truncation, I can change it to do that if that's preferred too